addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6631,27 +6630,29 @@ stock ParseFormula(boss, const String:key[], const String:defaultFormula[], defa
 	KvRewind(BossKV[Special[boss]]);
 	KvGetString(BossKV[Special[boss]], "name", bossName, sizeof(bossName), "=Failed name=");
 	KvGetString(BossKV[Special[boss]], key, formula, sizeof(formula), defaultFormula);
-	ReplaceString(formula, sizeof(formula), " ", "");  //Get rid of spaces
-
 	new bracket;  //Each bracket denotes a separate sum (within parentheses).  At the end, they're all added together to achieve the actual sum.
-	new Handle:sumArray=CreateArray(), Handle:_operator=CreateArray();
+	new Handle:sumArray=CreateArray(.startsize=1), Handle:_operator=CreateArray(.startsize=1);
```

Comment:
It fixes the odd chance I had when I made the eval Stock for SM where I ran into invalid index 0